collect_app/src/main/res/values/theme.xml

```diff
@@ -115,6 +115,9 @@
         <item name="android:statusBarColor">?colorSurface</item>
         <item name="android:windowLightStatusBar" tools:targetApi="m">@bool/lightStatusBar</item>
 
+        <!-- Disable edge-to-edge behavior -->
+        <item name="android:windowOptOutEdgeToEdgeEnforcement" tools:targetApi="35">true</item>
```

Comment:
I think we might need this more places. It looks like the status bar colors flicker when showing hiding a dialog, so I'm guessing `Theme.Collect.Dialog.Alert` needs this (or something similar) and so does any other theme we use (like `Theme.Collect.SplashScreen` for example).

Reply:
> It looks like the status bar colors flicker when showing hiding a dialog

Hmm, I don't see anything like that... any dialog? what android version are you using?

Reply:
Yup ignore me! This happens on Android 14 as well. It removes switches the status bar to light when applying the dialog "shade" if the device is in light mode. It appears to be standard Android behaviour, but just looks a little weird.